packages/app/src/components/Sliders/Slider.tsx

```diff
@@ -58,27 +58,31 @@ export function Slider(props: SliderProps) {
       <Show when={label()}>
         <span>{label()}</span>
       </Show>
-      <input
-        ref={(el) => {
-          scrollIntoViewAndFocusOnChange(value, el)
-        }}
-        class={ui.slider}
+      <div
```

Comment:
IOS/Safari fix for sliders, in their standard, input elements does not support `::before` classes so this wraps it, aligns everything and makes it cross browser compatible and quite identical in look and feel